tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -131,7 +131,7 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
 
       if (sourceFeature.hasTag("highway", "motorway_link", "trunk_link", "primary_link", "secondary_link",
         "tertiary_link")) {
-        feat.setAttr("pmap:link", 1).setZoomRange(12, 15);
+        feat.setAttrWithMinzoom("pmap:link", 1, 12);
```

Comment:
Suggest to:

>        feat.setAttr("pmap:link", 1);

Thanks for talking me thru this today @bdon.

To whit: In the post-processing step the `linkSimplify` function classifies OSM links into either "ramps" or "connectors". Ramps (at so-called clover leaf highway interchanges) are removed at low zooms and connectors (longer links between highways that are more like the start of the highway versus a ramp) are retained. This deserves some inline comments in the functions themselves!

So I think what we want is to _always_ set `pmap:link` with `setAttr` (not `setAttrWithMinzoom`)... and then let the  postProcess function's linkSimplify do it's magic.

I got confused because Tilezen doesn't have an equivalent `linkSimplify` function so uses a brute force method that's not as sophisticated (by setting the zoom ranges) and I reimplemented that instead of upgrading it.

NOTE: I've refactored this logic in my new PR so I'll need to resolve that conflict once this is merged. No big deal.